### PR TITLE
Add DIAMOND btop processing to get query & subject gaps

### LINKED ISSUE #439: Add DIAMOND btop processing
DIAMOND has added BLAST-like `btop` to its output format 6 directives. We need to parse it to find out how many gaps there are in the query and subject. See https://www.ncbi.nlm.nih.gov/books/NBK279682/ for details on the `btop` format.

### PR DESCRIPTION
**Pull request by terrycojones**
This needs some more tests, but you might want to have a look at it already.

Branched from #437.

Fixes #439.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
 ]
 
 setup(name='dark-matter',
-      version='1.0.50',
+      version='1.0.51',
```
Comment (bamueh) [inline setup.py]: This should be incremented, I think.

Comment (terrycojones) [inline setup.py]: Yes, thanks :-)  I remembered it when I was in bed, and then forgot it again.....  Done.

**Comment by bamueh** [comment]
Can you add a test for frame != 1 and with gaps (similar to the ones in `TestBlastxFramePlus1WithGaps`?

**Comment by terrycojones** [comment]
OK, I added `TestBlastxFramePlus2WithGaps` with the same 23 tests from `TestBlastxFramePlus1WithGaps`.

**Pull request merged**